test/contracts/external/median.spec.ts

```diff
@@ -0,0 +1,210 @@
+require('module-alias/register');
+
+import * as _ from 'lodash';
+import * as ABIDecoder from 'abi-decoder';
+import * as chai from 'chai';
+import * as setProtocolUtils from 'set-protocol-utils';
+import { Address, Bytes } from 'set-protocol-utils';
+import { BigNumber } from 'bignumber.js';
+
+import ChaiSetup from '@utils/chaiSetup';
+import { BigNumberSetup } from '@utils/bigNumberSetup';
+import {
+  FeedFactoryContract,
+  PriceFeedContract,
+  MedianContract,
+} from '@utils/contracts';
+import { expectRevertError } from '@utils/tokenAssertions';
+import { Blockchain } from '@utils/blockchain';
+import { ZERO } from '@utils/constants';
+import { getWeb3 } from '@utils/web3Helper';
+
+import { OracleWrapper } from '@utils/wrappers/oracleWrapper';
+
+BigNumberSetup.configure();
+ChaiSetup.configure();
+const web3 = getWeb3();
+const FeedFactory = artifacts.require('FeedFactory');
+const { SetProtocolTestUtils: SetTestUtils, SetProtocolUtils: SetUtils } = setProtocolUtils;
+const { expect } = chai;
+const blockchain = new Blockchain(web3);
+const setUtils = new SetUtils(web3);
+
+
+contract('Oracle', accounts => {
```

Comment:
Make this consistent with the contract name or an extension of it? `Median Oracle`?

Reply:
+1